notebooks/person-counting-webcam/README.md

```diff
@@ -0,0 +1,27 @@
+# Person Counting System using YOLOv8 and OpenVINO
+
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/person-counting-webcam/person-counting.ipynb)
+
+In this project, we utilized the YOLOv8 Object Counting class to develop a real-time person counting system using the YOLOv8 object detection model and tracking, optimized for Intel's OpenVINO toolkit to enhance inferencing speed. This system effectively monitors the number of individuals entering and exiting a room, leveraging the optimized YOLOv8 model for accurate person detection under varied conditions.
+
+By utilizing the OpenVINO runtime on Intel hardware, the system achieves significant improvements in processing speed, making it ideal for applications requiring real-time data, such as occupancy management and traffic flow control in public spaces and commercial settings.
+
+References:
+
+- YOLOv8 Object counting documentation: <a href="https://docs.ultralytics.com/guides/object-counting/" target="_blank">https://docs.ultralytics.com/guides/object-counting/</a>
+- OpenVINO Jupyter Notebooks: <a href="https://github.com/openvinotoolkit/openvino_notebooks/" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/</a>
+
+<div align="center"><img src="person-count.gif" width=900/></div>
```

Comment:
We are avoiding to posting files into repository explicitly, could you please use this link instead of person-count.gif ((https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/e0525f8a-4578-4c56-a0a5-ce68e30d2d45) and remove image from repo?